odata-parser.pegjs

```diff
@@ -258,8 +258,8 @@ FilterByExpressionLoop =
 
 			rhs:FilterByExpressionLoop
 			{
-				if (Array.isArray(lhs[0]) && op == lhs[0][0]) {
-					lhs[0] = [ op ].concat(lhs[0].slice(1), [rhs]);
+				if (Array.isArray(lhs[0]) && op === lhs[0][0]) {
+					lhs[0].push(rhs);
```

Comment:
This is the important change for performance, because it avoids a lot of memory allocation/copying